synapse/models/inet.py

```diff
@@ -825,62 +909,235 @@ def getModelDefs(self):
 
                     )),
 
-                    ('inet:whois:rar', {}, ()),
-
-                    ('inet:whois:reg', {}, ()),
+                    ('inet:web:action', {}, (
+                        ('act', ('str', {'lower': True}), {
+                            'doc': 'The action performed by the account.'
+                        }),
+                        ('acct', ('inet:web:acct', {}), {
+                            'ro': 1,
+                            'doc': 'The web account associated with the action.'
+                        }),
+                        ('acct:site', ('inet:fqdn', {}), {
+                            'ro': 1,
+                            'doc': 'The site or service associated with the account.'
+                        }),
+                        ('acct:user', ('inet:user', {}), {
+                            'ro': 1,
+                            'doc': 'The unique identifier for the account.'
+                        }),
+                        # FIXME missing json
```

Comment:
This was a get-out-of-jail-free card previously. I don't think this blocks merging but we'll eventually want to discuss it.

Reply:
Yep, I think figuring out how we want to index `json` type is a whole different task